nengo/builder/tests/test_transforms.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pytest
+
+from nengo.transforms import Convolution
+from nengo.builder.signal import Signal
+from nengo.builder.transforms import ConvInc
+
+
+@pytest.mark.parametrize("channels_last", (True, False))
+@pytest.mark.parametrize("stride0", (1, 2))
+@pytest.mark.parametrize("stride1", (1, 2))
+@pytest.mark.parametrize("kernel0", (4, 5))
+@pytest.mark.parametrize("kernel1", (4, 5))
+@pytest.mark.parametrize("shape0", (16, 17))
+@pytest.mark.parametrize("shape1", (16, 17))
+@pytest.mark.parametrize("padding", ("same", "valid"))
```

Comment:
Not sure if we're worried about time for the nengo tests, but this consists of 256 tests and adds 25 seconds to the test suite. It's good that it's thorough, but I'm wondering if there's a way to make it a little shorter at least for the typical tests. We could still keep more extended tests that we run manually if someone's making changes to the conv code.

Reply:
Note that this is already automatically skipped in the typical tests on TravisCI, as it is skipped if scipy isn't installed.  So it's only an issue for the one run we do with scipy, or for people testing locally.  But we also have a pytest marker for slow tests, if we wanted to add an additional opt-in layer.

Reply:
I also reduced this to 64 conditions now.

Reply:
Yeah, I might make them opt-in. It would be nice if one of the CI tests ran them, though. Would it be possible to make the Scipy test-run opt in to slow tests as well? I know there's been at least one time in the past when a slow test has been broken in master and we didn't notice because they don't always get run. Or would adding those to the Scipy run put us at risk of timing out? @tbekolay